charts/testkube-api/templates/deployment.yaml

```diff
@@ -223,14 +223,22 @@ spec:
             - name: http
               containerPort: 8088
               protocol: TCP
-          livenessProbe:
-            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          startupProbe:
+            initialDelaySeconds: 3
+            periodSeconds: 3
+            timeoutSeconds: 3
+            failureThreshold: 20
```

Comment:
guess, better to avoid hard coded values

Reply:
Not sure, I do not really see use cases where these values should be overridden 🤔 Want to avoid overengineering this and follow YAGNI (You Aren't Gonna Need It)